substate/substate.go

```diff
@@ -79,3 +80,23 @@ func (s *Substate) String() string {
 
 	return builder.String()
 }
+
+func (s *Substate) Dump(block uint64, tx int) error {
```

Comment:
I think we would better do with `String()`method
https://go.dev/tour/methods/17

Reply:
`Dump()` deprecated in favor of `String()`

[e655a88](https://github.com/Fantom-foundation/Substate/pull/85/commits/e655a885fc7cfb7f23fa9d90f5d92f14d330f109)